Library/Formula/nedit.rb

```diff
@@ -0,0 +1,39 @@
+class Nedit < Formula
+  homepage "http://www.nedit.org"
+  url "https://downloads.sourceforge.net/project/nedit/nedit-source/nedit-5.6-src.tar.gz"
+  sha1 "133641224b0969e31e3404b874c71c35341f028b"
+
+  depends_on :x11
+  depends_on "openmotif"
+
+  def install
+
+    inreplace "util/motif.c" do |s|
+      s.gsub! /200303/, "200304"
+    end
+
+    args = %W[
+      MOTIFLINK='-lXm'
+    ]
+
+    system "make", "macosx", *args
+    mv "source/nc", "source/ncl"
+
+    system "make -C doc man doc"
```

Comment:
The first doc is the value for the -C argument of make. It corresponds to the nedit-5.6/doc directory in the unpacked source tree. So, whereas it is not normal to have doc there twice, it is necessary, and logical in this case. :)